src/main/api/2/studio-api-2.yaml

```diff
@@ -1115,6 +1119,159 @@ paths:
         '500':
           $ref: '#/components/responses/InternalServerError'
 
+  /cluster:
+    get:
+      tags:
+        - cluster
+      summary: Get all the members of the cluster
+      description: 'Required Permission: "READ_CLUSTER"'
+      operationId: getClusterMembers
+      responses:
+        '200':
+          description: OK
+          content:
+            application/json:
+              schema:
+                type: object
+                properties:
+                  result:
+                    type: object
+                    properties:
+                      response:
+                        $ref: '#/components/schemas/ApiResponse'
+                      entities:
+                        type: array
+                        items:
+                          $ref: '#/components/schemas/ClusterMember'
+        '400':
+          $ref: '#/components/responses/BadRequest'
+        '401':
+          $ref: '#/components/responses/Unauthorized'
+        '404':
+          $ref: '#/components/responses/NotFound'
```

Comment:
I don't think we need to throw this error?

Reply:
What should we return if there is no cluster (single node with the DB table empty)?

Reply:
You're right we should return 404